tests/test_pipelines.py

```diff
@@ -220,6 +246,24 @@
     "isolated_star_sources",
 }
 
+# Outputs common to all "quickLook" pipelines, which only iclude
+QUICKLOOK_OUTPUTS = {
+    "calexp", "visitSummary"
+}
+
+# All refcats used by any pipelines, for inclusion as the initial_dataset_types
+# argument to PipelineStepTester; the intent is just to allow the 'skypix'
+# dimension placeholder used in these connections to be resolved.
+REFCATS = [
+    ("ps1_pv3_3pi_20170110", {"htm7"}, "SimpleCatalog", False),
+    ("gaia_dr2_20200414", {"htm7"}, "SimpleCatalog", False),
+    ("gaia_dr3_20230707", {"htm7"}, "SimpleCatalog", False),
```

Comment:
I think it's okay to have this here even though we don't have all these refcats on the summit.

Reply:
Though do we need a subset that is on the summit and a fuller set that is at DFs and make sure that quicklook uses only summit refcats?

Reply:
This is just a convenience list of all the refcats anywhere (note that it includes simulated ones).  The real check on which refcat is used in a pipeline is the variables used to supply `expected_inputs` to the tester.

Reply:
Ah!  I see.  Then my comment above that gaia_dr3 should be removed from the lsst inputs.